src/main/java/com/eprosima/fastcdr/idl/templates/FastCdrCommon.stg

```diff
@@ -14,6 +14,34 @@
 
 group FastCdrCommon;
 
+paramRetType(typecode) ::= <%
+$if(typecode)$
+$typecode.cppTypename$
+$else$
+void
+$endif$
+%>
+
+paramTypeByRef(typecode) ::= <%
+$typecode.cppTypename$&
+%>
+
+paramTypeByValue(typecode, feed) ::= <%
+$if(feed)$
+std::shared_ptr<eprosima::fastdds::dds::rpc::RpcClientWriter<$typecode.cppTypename$> >&
+$else$
+$if(typecode.primitive)$
+$typecode.cppTypename$
+$else$
+const $typecode.cppTypename$&
+$endif$
+$endif$
+%>
+
+paramDeclarations(params, initialSeparator="") ::= <<
```

Comment:
Maybe use `<%` instead of `<<`?

Reply:
I think we want each argument on a different line. E.g.:

```
    virtual eprosima::fastdds::dds::rpc::RpcFuture<void> representation_limits(
            /*out*/ int32_t& min_value,
            /*out*/ int32_t& max_value) = 0;

    virtual eprosima::fastdds::dds::rpc::RpcFuture<int32_t> addition(
            /*in*/ int32_t value1,
            /*in*/ int32_t value2) = 0;
```